charts/kof-storage/templates/victoria/vmauth.yaml

```diff
@@ -17,5 +18,25 @@ spec:
   license: {}
   port: "8427"
   selectAllByDefault: true
+
+  # Protecting from CVE-2019-11248 in VMAuth which exposes `/debug/pprof`
+  # and a lot of other endpoints listed in https://docs.victoriametrics.com/#security
+  {{- if not .Values.global.lint }}
```

Comment:
Why do we need this condition? It is needed only for helm functions like lookup that don't work during linting